src/styles/main.scss

```diff
@@ -1,3 +1,355 @@
+  font-weight: 400;
+  line-height: 140%;
+}
+
+.products {
+  margin-top: 64px;
+}
+
+.products__item {
+  margin-top: 40px;
+}
+
+.products__link {
+  display: flex;
+}
+
+.products__link--img {
+  margin-top: 40px;
+}
+
+.products__price {
+  margin: 0;
+  margin-top: 12px;
+  color: #292929;
+  font-size: 16px;
+  font-family: Space Grotesk, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 140%;
+}
+
+/* #endregion */
+/* region store */
+.store {
+  margin-top: 96px;
+}
+
+.store_section--img {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 40px;
+}
+
+.products__subtitle {
+  margin: 0;
+  margin-top: 16px;
+  color: #292929;
+  font-family: Space Grotesk, sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 140%;
+}
+
+.products__btn {
+  margin: 0;
+  margin-top: 40px;
+  width: 280px;
+  height: 56px;
+  padding: 0 48px;
+  border-radius: 8px;
+  border: none;
+  background: #292929;
+  color: #fff;
+  text-align: center;
+  font-size: 16px;
+  font-family: Space Grotesk, sans-serif;
+  font-weight: 700;
+  line-height: 27px;
+}
+/* endregion */
+
+/* #region about-us */
+.about-us {
+  margin-top: 144px;
+}
+
+.about-us_info {
+  margin-top: 16px;
+  color: #292929;
+  font-size: 16px;
+  font-family: Space Mono, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 140%;
+}
+/* #endregion */
+
+/* #region contuct us */
+.contacts {
+  margin-top: 104px;
+}
+
+.contacts__form {
+  margin-top: 26px;
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+}
+
+.form__field {
+  padding: 16px 14px;
+  margin-top: 16px;
+  border: none;
+  border-radius: 8px;
+  background: #f8f8fa;
+  color: #7e7e83;
+  font-size: 14px;
+  font-family: Space Mono, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 140%;
+}
+
+.form__btn {
+  margin-top: 32px;
+  height: 56px;
+  border-radius: 8px;
+  border: 2px solid #f0f1f4;
+  background: #f8f8fa;
+  color: #292929;
+  text-align: center;
+  font-size: 16px;
+  font-family: Space Grotesk, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 27px;
+}
+
+.contacts__block {
+  margin-top: 48px;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.contacts__item {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.contacts__item--title {
+  margin: 0;
+  color: #9393a3;
+  font-size: 14px;
+  font-family: Space Grotesk, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 140%;
+}
+
+.contacts__item--link {
+  text-decoration: none;
+  color: #191919;
+  font-size: 16px;
+  font-family: Space Mono, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 140%;
+}
+/* #endregion */
+/* #region footer */
+.footer {
+  margin-top: 96px;
+  height: 200px;
+  background-size: cover;
+  background-position: center;
+  background-image: url(../images/footer_bg_mobile.png);
+  position: sticky;
```

Comment:
The footer element has a position of 'sticky'. Therefore, the footer remains on the screen constantly during scrolling, which looks incorrect.
https://gyazo.com/483ce0daa68de240efb9d511a82b2208

Also, the image has the wrong formatting according to the design.